src/index.ts

```diff
@@ -252,18 +259,14 @@ export { SiteAggregates } from "./types/SiteAggregates";
 export { SiteId } from "./types/SiteId";
 export { SiteResponse } from "./types/SiteResponse";
 export { SiteView } from "./types/SiteView";
-export { SortType } from "./types/SortType";
 export { SubscribedType } from "./types/SubscribedType";
 export { SuccessResponse } from "./types/SuccessResponse";
 export { Tagline } from "./types/Tagline";
+export { TaglineId } from "./types/TaglineId";
+export { TaglineResponse } from "./types/TaglineResponse";
 export { TransferCommunity } from "./types/TransferCommunity";
+export { UpdateTagline } from "./types/UpdateTagline";
 export { UpdateTotp } from "./types/UpdateTotp";
 export { UpdateTotpResponse } from "./types/UpdateTotpResponse";
 export { VerifyEmail } from "./types/VerifyEmail";
 export { VoteView } from "./types/VoteView";
-export {
-  UploadImage,
-  UploadImageResponse,
-  ImageFile,
-  DeleteImage,
-} from "./other_types";
```

Comment:
This block needs to go back in. Its an imperfection in our script, that unfortunately requires manually restoring. I'm not sure what the best way to handle that would be, but for now we just have to add it back in manually.